src/js/components/tablero/GraphCard.jsx

```diff
@@ -46,6 +47,8 @@ const GraphCard = SortableElement(({
     />);
   } else if (cardType === 'numbers') {
     graph = <Numbers data={data} options={options} />;
+  } else if (cardType === 'numbersCustomColors') {
+    graph = <NumbersRAG data={data} options={options} />;
```

Comment:
I don't see options property being used in the NumbersRAG component.

Reply:
What about this parameter?

Reply:
It is true, I change it right now

Reply:
I just deleted it